pom.xml

```diff
@@ -171,6 +171,9 @@
 				<configuration>
 					<archive>
 						<manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
+						<manifestEntries>
+							<Automatic-Module-Name>org.gitlab4j</Automatic-Module-Name>
```

Comment:
Where is the best practice definition for automatic-module-names?

since our *groupId* is: `org.gitlab4j`
and our *artifactId* is: `gitlab4j-api`

I am wondering if it wouldn't be better to name this module `org.gitlab4j.api`…

Reply:
I don't know. Your suggestion `org.gitlab4j.api` absolutely sounds reasonable.

Reply:
@jabby do you have an opinion?

Reply:
I think `org.gitlab4j.api` is a good idea.

Reply:
> I think `org.gitlab4j.api` is a good idea.

Updated to `org.gitlab4j.api`.